certora/specs/Reverts.spec

```diff
@@ -0,0 +1,253 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+import "ConsistentInstantiation.spec";
+
+using Morpho as MORPHO;
+
+methods {
+    function _.position(PreLiquidation.Id, address) external => DISPATCHER(true);
+    function _.accrueInterest(PreLiquidation.MarketParams) external => DISPATCHER(true);
```

Comment:
why is that dispatched? I don't get it

Reply:
This avoids unexpected havocks.

Reply:
I think we should investigate why it havocs, and if it does not make sense let's open a ticket with the Certora team

Reply:
Good idea!

Reply:
Manged to remove the in https://github.com/morpho-org/pre-liquidation/pull/37/commits/ebdaff6e0cdf9a9030237d530d03bf9708364178.